site/app/controllers/OfficeHoursQueueController.php

```diff
@@ -452,6 +452,29 @@ public function changeRegex($queue_code) {
         );
     }
 
+    /**
+     * @Route("/courses/{_semester}/{_course}/office_hours_queue/{queue_code}/change_contact_information", methods={"POST"})
+     * @AccessControl(role="LIMITED_ACCESS_GRADER")
+     * @return MultiResponse
+     */
+    public function changeContactInformation($queue_code) {
+        if (empty($queue_code)) {
+            $this->core->addErrorMessage("Missing queue name");
+            return MultiResponse::RedirectOnlyResponse(
```

Comment:
Can just do `return new RedicrectResponse(...)` instead of calling the multiresponse since we've deprecated that function

Reply:
> There's a lot of code related to gradeable prerequisite logic, should that be removed from this branch?

Everything should be good now, can you please take another look when you get a chance?